tests/test_isolanguage.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016-2020 CERN.
+#
+# Marshmallow-Utils is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test the marshmallow ISO Language."""
+
+import pytest
+from marshmallow import Schema, ValidationError, fields
+
+from marshmallow_utils.fields import isolanguage
```

Comment:
Can import `ISOLangString` directly

```suggestion
from marshmallow_utils.fields import ISOLangString
```